internal/provider/provider_test.go

```diff
@@ -62,11 +64,19 @@ func setupVCR(t *testing.T, cassetteName string) *recorder.Recorder {
 	hookRedactIntegrationUserCredentials := func(i *cassette.Interaction) error {
 		intUser := os.Getenv("BTP_USERNAME")
 		intUserPwd := os.Getenv("BTP_PASSWORD")
+		intUserIdp := os.Getenv("BTP_IDP")
 
 		firstName, lastName := getNameFromEmail(intUser)
 
 		if strings.Contains(i.Request.URL, "/login/") {
 			i.Request.Body = strings.ReplaceAll(i.Request.Body, intUserPwd, "redacted")
+			i.Request.Body = strings.ReplaceAll(i.Request.Body, "\"customIdp\":\""+intUserIdp+"\"", "\"customIdp\":\"\"")
+			i.Response.Body = strings.ReplaceAll(i.Response.Body, "\"issuer\":\""+intUserIdp+"\"", "\"issuer\":\"accounts.sap.com\"")
+			i.Response.Body = strings.ReplaceAll(i.Response.Body, "\"issuer\":\""+intUserIdp+".accounts400.ondemand.com\"", "\"issuer\":\"accounts.sap.com\"")
+		}
+
+		if _, exists := i.Request.Headers[http.CanonicalHeaderKey(btpcli.HeaderCLICustomIDP)]; exists {
+			i.Request.Headers.Set(btpcli.HeaderCLICustomIDP, "")
```

Comment:
Shouldn't this be set to the modified idp instead of empty string?

Reply:
To match fixtures recorded without idp, I decided to set this to empty string (see https://github.com/SAP/terraform-provider-btp/pull/247#issuecomment-1611571896). What we could do is to always replace this with a dummy value, but that would require a re-recording of all fixtures.

Reply:
Fair enough. May I ask you to leave a comment for this?

Reply:
Good point. Then I'll leave a TODO for later changing this into a test value.